src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/DecompressionHandler.cs

```diff
@@ -159,7 +160,7 @@ private async ValueTask<Stream> CreateContentReadStreamAsyncCore(bool async, Can
                 }
                 else
                 {
-                    originalStream = _originalContent.ReadAsStream();
+                    originalStream = _originalContent.ReadAsStream(cancellationToken);
```

Comment:
Forgot to add this other case, @ManickaP

Reply:
It's OK here as well.